tutorials/langchain_pinecone_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
<p>Nice! Feels like we should embed this in our docs too!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>